tests/commonTemplates_test.go

```diff
@@ -331,4 +334,79 @@ var _ = Describe("Common templates", func() {
 			}
 		})
 	})
+
+	Context("rbac", func() {
+		Context("os-images", func() {
+			var (
+				regularSA *core.ServiceAccount
+			)
+
+			BeforeEach(func() {
+				regularSA = &core.ServiceAccount{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "regular-sa",
+						Namespace: strategy.GetNamespace(),
+					},
+				}
+
+				Expect(apiClient.Create(ctx, regularSA)).ToNot(HaveOccurred(), "creation of regular service account failed")
+				Expect(apiClient.Get(ctx, getResourceKey(regularSA), regularSA)).ToNot(HaveOccurred())
```

Comment:
Calling `Get` after `Create` is not needed. The `Create` API call returns the created object and populates it into `regularSA` variable, similarly to `Get`.